guides/user/kubernetes-v2/deploy-helm/index.md

```diff
@@ -14,6 +14,12 @@ with the help of a templating engine. Currently, the only supported templating
 engine is [Helm](https://helm.sh/), by relying on the `helm template` command.
 See more details [here](https://docs.helm.sh/helm/#helm-template).
 
+> Note: This stage is intended to help you package and deploy applications 
+> that you own, and are actively developing and redeploying frequently.
+> It is not intended to serve as a one-time installation method for
+> third-party packages. If that is your goal, it's arguably better to call
+> `helm apply` once when bootstrapping your Kubernetes cluster.
+
```

Comment:
@lwander Is there anything you could link to for more information?

Reply:
More information about what?

Reply:
I was thinking `helm apply`, but I don't find it on [helm.sh](https://helm.sh/).

Reply:
Good call, I was even pointing out the wrong command for the version of helm we support